<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentUi</class>
 <widget class="QMainWindow" name="StudentUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="coursesBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>801</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QListView" name="listViewCourses">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Courses</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_new_course_button">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>420</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Register in Course</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_close_button">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>2000</y>
       <width>110</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Save and Close</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>481</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Course ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="course_Id"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="course_name"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Department</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="course_department"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Schedule</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="course_schedule"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Pre-Requisits</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="course_prerequists"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Instructor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="course_instructor"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Status</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="course_status"/>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Deadline</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="course_deadline"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="coursesBtn">
       <property name="text">
        <string>Courses</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="studentsBtn">
       <property name="text">
        <string>Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="eventsBtn">
       <property name="text">
        <string>Events</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="profileBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>801</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Profile and Notifications</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_close_button_2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>2000</y>
       <width>110</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Save and Close</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>751</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Student Id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="student_id"/>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="student_name"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="student_email"/>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="student_password"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Academic year</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="student_academic_year"/>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Enrolled courses</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="student_enrolled_courses"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Completed courses</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="student_complete_courses"/>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Registered Events</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="student_registered_events"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>357</y>
       <width>751</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Upcomming events deadline</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="student_event_deadline"/>
      </item>
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Upcomming courses dealine</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="student_courses_deadline"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="eventsBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QListView" name="listViewEvents">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Events</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_new_course_button_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>420</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Register in Event</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_close_button_3">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>2000</y>
       <width>110</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Save and Close</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>481</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Event Id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="event_id"/>
        </item>
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="event_title"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="event_description"/>
        </item>
        <item>
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Schedule</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="event_schedule"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Speaker</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="event_speaker"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
